src/include/catalog/catalog_defs.h

```diff
@@ -5,5 +5,8 @@
 namespace terrier::catalog {
 STRONG_TYPEDEF(col_oid_t, uint32_t);
 STRONG_TYPEDEF(db_oid_t, uint32_t);
+STRONG_TYPEDEF(index_oid_t, uint32_t);
+STRONG_TYPEDEF(indexkeycol_oid_t, uint32_t);
+STRONG_TYPEDEF(sqltable_oid_t, uint32_t);
```

Comment:
@mbutrovich Are we using this? Code search not working.

Reply:
It's not really in the scope of this PR. I'll see if it makes sense to revert some.